src/iframeResizer.js

```diff
@@ -168,21 +168,20 @@
     }
 
     function getPaddingEnds(compStyle) {
-      
       if (compStyle.boxSizing !== 'border-box') {
         return 0;
       }
-      var top = compStyle.paddingTop ? parseInt(compStyle.paddingTop, 10) : 0
-      var bot = compStyle.paddingBottom ? parseInt(compStyle.paddingBottom, 10) : 0
+      var top = compStyle.paddingTop ? parseInt(compStyle.paddingTop.replace('px', ''), 10) : 0
+      var bot = compStyle.paddingBottom ? parseInt(compStyle.paddingBottom.replace('px', ''), 10) : 0
       return top + bot
     }
     
     function getBorderEnds(compStyle) {
       if (compStyle.boxSizing !== 'border-box') {
         return 0;
       }
-      var top = compStyle.borderTopWidth && compStyle.borderTopWidth.indexOf('px') !== -1 ? parseInt(compStyle.borderTopWidth.replace('px', ''), 10) : 0
-      var bot = compStyle.borderBottomWidth && compStyle.borderBottomWidth.indexOf('px') !== -1 ? parseInt(compStyle.borderBottomWidth.replace('px', ''), 10) : 0
+      var top = compStyle.borderTopWidth ? parseInt(compStyle.borderTopWidth.replace('px', ''), 10) : 0
```

Comment:
I don’t think you need `replace` here, as `parseInt('10px') === 10`

Reply:
Oh, right. I didn't realise the 'px' got stripped out automatically. I've pushed an update.
I want to leave the radix (10) in there though, as mozilla docs recommend _"always specify a radix when using parseInt"_ because of some browsers defaulting to other values (it doesn't specify which browsers though unfortunately) - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
Hi David. Was there some reason this hasn't been approved yet? Happy to discuss.